src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -203,15 +203,13 @@ public function renderViewElement(
      *
      * @param mixed $baseObject
      * @param mixed $compareObject
```

Comment:
You should remove this to avoid phpstan error

Reply:
Thanks. There are different parameters and return types which are mixed etc, but I left those as they are. Not sure what's good practice in those cases, just to document them in phpdoc block? Maybe I'll g add missing iterable value types and method descriptions at some point in future if I get more familiar with the code and now to with adding just type hints so to move forward with the tasks needed for S5.

Reply:
I'm not sure to understand.

When mixed it the valid value (because we accept everything) we should use mixed in phpdoc.
When it's not, (like in your case, you added `object` typehint)  we should not use mixed.

Some `mixed` are wrong, and we try to fix them with these PR about param/return typehint.